articles/azure-stack/azure-stack-monitor-update.md

```diff
@@ -21,7 +21,7 @@ ms.author: mabrigg
 
 *Applies to: Azure Stack integrated systems*
 
-You can use the privileged endpoint to monitor the progress of an Azure Stack update run, and to resume a failed update run from the last successful step. 
+You can use the privileged endpoint to monitor the progress of an Azure Stack update run, and to resume a failed update run from the last successful step should the Azure Stack portal become unavailable.  Using the Azure Stack portal is the recommended method to manage updates in Azure Stack.
```

Comment:
This looks good.

Reply:
Thanks Matt, is everything good to publish now?